src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/SuggestionIconKind.cs

```diff
@@ -9,5 +9,6 @@ public enum SuggestionIconKind
         Function,
         Control,
         DataSource,
+        Variable
```

Comment:
>Variable [](http://example.com/codeflow?start=8&length=8)

I'm not sure the impact of adding to this enum. 
For now, can we stick to Other?